luzer/init.lua

```diff
@@ -26,7 +26,9 @@ local function build_flags(arg, func_args)
     local flags = {}
     for _, arg_str in ipairs(arg) do
         local name, value
-        if starts_with(arg_str, "-") then
+        -- XXX: Ignore the symbol `--` because in the OSS Fuzz it
```

Comment:
This means that `--` is considered as a `corpus` value, which is not quite true.

Reply:
Minor: "2 runs" looks like a bug, but it is out of scope of this issue. Maybe it is better to use `\d` instead to avoid changing this test later if we fix the runs flag?